src/main/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommand.java

```diff
@@ -0,0 +1,83 @@
+package seedu.wife.logic.commands.tagcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.wife.commons.core.Messages;
+import seedu.wife.commons.core.index.Index;
+import seedu.wife.logic.commands.Command;
+import seedu.wife.logic.commands.CommandResult;
+import seedu.wife.logic.commands.exceptions.CommandException;
+import seedu.wife.model.Model;
+import seedu.wife.model.food.Food;
+import seedu.wife.model.tag.Tag;
+
+/**
+ * Command to untag a food item.
+ */
+public class UntagFoodCommand extends Command {
+    public static final String COMMAND_WORD = "untag";
+    public static final String MESSAGE_USAGE = "Remove a tag from a food item in your list.\n"
+            + "Format:\n"
+            + "untag <index> n/<tag name>\n"
+            + "example: untag 1 n/Vege";
+    private Tag tag;
+    private Index index;
+
+    /**
+     * Constructor to create a new UntagFoodCommand object.
+     */
+    public UntagFoodCommand(String tagName, Index index) {
+        requireNonNull(tagName);
+        requireNonNull(index);
+        this.tag = new Tag(tagName);
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        Food foodToUntag = UntagFoodCommand.getFoodToUntag(model, tag, index);
```

Comment:
Will need to have a `requireNonNull` check for the model

Reply:
done!